README.md

```diff
@@ -84,7 +97,7 @@ On the other hand, the **client application** is responsible to do the following
 
 ## How do I build my own state channel demo?
 
-You may take a look at Helpful Links Section, there is a State Channel Demo Tutorial Guide.
+You can find the steps at the complete [State Channel Demo Tutorial Guide](TUTORIAL.md).
 
 ## How do State Channel Demo Game transactions take place?
 
```

Comment:
Re. the "in order to avoid cheating in a turn-based implementation ...", you strictly don't need to hash the move. Forced progress can be used to ensure that the opponent accepts the move (and the contract should ensure that moves are actually made in turn). What makes this case (rock-paper-scissors) special is rather that the players are supposed to move _at the same time_, whereas the contract approach enforces a turn-based approach.

Perhaps this is what was meant with "in a turn-based implementation"? It's not a turn-based game, but the implementation makes it so, necessitating the information hiding.

Reply:
This is exactly what was meant with "in a turn-based implementation". Do you have any suggestions on if/how we should rephrase this?

Reply:
I think this is more an explanation by Ulf that we have a special case here in the demo where hashing is needed as both parties need to perform the action at the same time.

if we take "4 wins" as an example a hash wouldn't be needed. the opponent would see the move and just refuse to co-sign the winning move. in this case you could force progress your contract via on-chain force progress.

wondering if I should add this example 🤔 I am updating the readme / tutorial anyway right now